Assets/Group1/Scripts/EnemiesCounter.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemiesCounter : MonoBehaviour
+{
+    [SerializeField] private GameObject _blackPanel;
+
+    private HashSet<Enemy> _enemies = new HashSet<Enemy>();
+
+    private float emptySet = 0;
+
+    private void Start()
+    {
+        foreach (Enemy enemy in FindObjectsOfType<Enemy>())
+        {
+            enemy.OnDead += ObjectDeadHandler;
+            _enemies.Add(enemy);
+        }
+    }
+
+    private void ObjectDeadHandler(Enemy enemy)
```

Comment:
https://trello.com/c/CsOZfMhi/1-%D0%BD%D0%B0%D0%B7%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9-%D0%BE%D0%B1%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D1%87%D0%B8%D0%BA%D0%BE%D0%B2